phpseclib/Math/BigInteger/Engines/PHP/Base.php

```diff
@@ -80,7 +80,7 @@ public static function isValidEngine()
      * @param string $class
      * @return \phpseclib\Math\BigInteger\Engines\PHP
      */
-    protected static function powModHelper(PHP $x, PHP $e, PHP $n, $class)
+    public static function powModHelper(PHP $x, PHP $e, PHP $n, $class)
```

Comment:
File PHP.php
method:

protected function powModInner(PHP $e, PHP $n)
{
try {
$class = static::$modexpEngine;
return $class::powModHelper($this, $e, $n, static::class);
} catch (\Exception $err) {
return PHP\DefaultEngine::powModHelper($this, $e, $n, static::class);
}
}

As you can see here's a static call which can't be performed if the powModHelper method is not public